src/engine/QueryExecutionContext.h

```diff
@@ -32,20 +32,16 @@ using std::vector;
 class CacheValue {
  private:
   std::shared_ptr<const ResultTable> _resultTable;
-  RuntimeInformation _runtimeInfo;
 
  public:
-  explicit CacheValue(ResultTable resultTable, RuntimeInformation runtimeInfo)
+  explicit CacheValue(ResultTable resultTable)
```

Comment:
I think we still need this!
Otherwise we will lose information about the original computation costs of the cached trees. Those are really valuable for analytics!